stories/wizard/wizard.stories.js

```diff
@@ -0,0 +1,897 @@
+| 2rem | &nbsp;&nbsp;&nbsp;&nbsp;600px | &nbsp;&nbsp;&nbsp;&nbsp;1023px | &nbsp;&nbsp;&nbsp;&nbsp;<code>fd-wizard\\_\\_progress-bar--m</code> / <code>fd-wizard\\_\\_content--m</code> |
+| 2rem | &nbsp;&nbsp;&nbsp;&nbsp;1024px | &nbsp;&nbsp;&nbsp;&nbsp;1439px | &nbsp;&nbsp;&nbsp;&nbsp;<code>fd-wizard\\_\\_progress-bar--lg</code> / <code>fd-wizard\\_\\_content--lg</code> |
+| 3rem | &nbsp;&nbsp;&nbsp;&nbsp;1440px | &nbsp;&nbsp;&nbsp;&nbsp;_n/a_ | &nbsp;&nbsp;&nbsp;&nbsp;<code>fd-wizard\\_\\_progress-bar--xl</code> / <code>fd-wizard\\_\\_content--xl</code> |
+
+
+## Modifiers
+
+| **Steps** | &nbsp;&nbsp;&nbsp;&nbsp; **Modifier class** |
+| :---- | :-------------- |
+| Completed     |  &nbsp;&nbsp;&nbsp;&nbsp; <code>fd-wizard\\_\\_step--completed</code>     |
+| Current       |  &nbsp;&nbsp;&nbsp;&nbsp; <code>fd-wizard\\_\\_step--current</code>       |
+| Upcoming      |  &nbsp;&nbsp;&nbsp;&nbsp; <code>fd-wizard\\_\\_step--upcoming</code>      |
+| No-label      |  &nbsp;&nbsp;&nbsp;&nbsp; <code>fd-wizard\\_\\_step--no-label</code>      |
+| Stacked       |  &nbsp;&nbsp;&nbsp;&nbsp; <code>fd-wizard\\_\\_step--stacked</code>       |
+| Stacked on top   |  &nbsp;&nbsp;&nbsp;&nbsp; <code>fd-wizard\\_\\_step--stacked-top</code>   |
+| Active        |  &nbsp;&nbsp;&nbsp;&nbsp; <code>fd-wizard\\_\\_step--active</code>        |
+
+## Connector types
+
+There are multiple connector types that can be displayed depending on the steps involved.
+
+
+| **Connector type** | &nbsp;&nbsp; &nbsp;&nbsp; **Connection** | &nbsp;&nbsp; &nbsp;&nbsp; **Class/Modifier** |
+| :---- | :-------------- | :-------------- |
+| Default | &nbsp;&nbsp; &nbsp;&nbsp; Inactive step (or Active step) to Inactive step |  &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_connector</code> |
+| Active | &nbsp;&nbsp; &nbsp;&nbsp; Active step to Active step | &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_connector--active</code> |
+| Branching | &nbsp;&nbsp; &nbsp;&nbsp; Branching step to Inactive step or no step | &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_connector--branching</code> |
+
+## Truncation rules
+
+| **Type** | &nbsp;&nbsp; &nbsp;&nbsp; **Max lines** | &nbsp;&nbsp; &nbsp;&nbsp; **Modifier class** |
+| :---- | :-------------- | :-------------- |
+| Label | &nbsp;&nbsp; &nbsp;&nbsp; 2 |  &nbsp;&nbsp; &nbsp;&nbsp; Text truncates when it exceeds the maximum space available. When a step has optional text, the label can only accommodate 1 line. |
+| Label with optional text | &nbsp;&nbsp; &nbsp;&nbsp; 1 | &nbsp;&nbsp; &nbsp;&nbsp; Text truncates when it exceeds the maximum space available. The <code>fd-wizard\\_\\_label-container--optional</code> modifier class is applied together with <code>fd-wizard\\_\\_label-container</code>. |
+| Optional text | &nbsp;&nbsp; &nbsp;&nbsp; 1 | &nbsp;&nbsp; &nbsp;&nbsp; Text truncates when it exceeds the maximum space available. |
```

Comment:
So far, all of the tables look fantastic! I tried making the window smaller to see how they all looked, and I think here it's the only place where the text wraps too soon (the **Max lines** column). I'm not sure if more spacing would help here or if it would look more jarring. But I figured I would point this out, I can get to it later and test out some things. :)